docs/docs/en/development/development-environment-setup.md

```diff
@@ -5,10 +5,9 @@
 Before setting up the DolphinScheduler development environment, please make sure you have installed the software as below:
 
 * [Git](https://git-scm.com/downloads): DolphinScheduler version control system
-* [JDK](https://www.oracle.com/technetwork/java/javase/downloads/index.html): DolphinScheduler backend language
+* [JDK](https://www.oracle.com/technetwork/java/javase/downloads/index.html): DolphinScheduler backend language, have to use version after JDK 1.8
 * [Maven](http://maven.apache.org/download.cgi): Java Package Management System
-* [Node](https://nodejs.org/en/download): DolphinScheduler frontend
- language
+* [Node12.20.2](https://nodejs.org/en/download): DolphinScheduler frontend
```

Comment:
You should change `Node` as you do to JDK

Reply:
I have modified the documentation